include/simdjson/generic/ondemand/value-inl.h

```diff
@@ -404,7 +403,7 @@ simdjson_inline simdjson_result<const char *> simdjson_result<SIMDJSON_IMPLEMENT
   return first.current_location();
 }
 
-simdjson_inline int32_t simdjson_result<SIMDJSON_IMPLEMENTATION::ondemand::value>::current_depth() const noexcept {
+simdjson_inline simdjson_result<int32_t> simdjson_result<SIMDJSON_IMPLEMENTATION::ondemand::value>::current_depth() const noexcept {
   if (error()) { return error(); }
```

Comment:
Was this coercing the error_code to an int32_t before the change? Not familiar with operator overloading (noexcept is used)

Reply:
"Was this coercing the error_code to an int32_t before the change?"

Yes. It was a bug.